### DISCUSSION TITLE
How i disable CORS in new Stable?

### DISCUSSION THREAD

**Discussion by pedrolaxe** [strapi/strapi#6481] [Legacy Topics]
Hey Guys, How i configure CORS in new version 3.0 i tried configure in ./config/middlewares.js but i not successful.

**Comment by asdutoit** [comment]
your js file should be named:  `middleware.js` not `middlewares.js`.
I think the file should also contain the following:

```
module.exports = {
  //...
  settings: {
    cors: {
      enabled: false,
    },
  },
};

```

**Reply by pedrolaxe** [reply]
I Add this file with configurations, but not working.. the CORS already is blocking.
